twoliter/embedded/build.Dockerfile

```diff
@@ -181,6 +181,11 @@ ARG KIT
 ARG PACKAGE_DEPENDENCIES
 ARG ARCH
 ARG NOCACHE
+ARG BUILD_ID
+ARG VERSION_ID
+ARG EXTERNAL_KIT_METADATA
+ARG VENDOR
+ARG LOCAL_KITS
```

Comment:
This should be `KIT_DEPENDENCIES` or `LOCAL_KIT_DEPENDENCIES` to align with the naming of `PACKAGE_DEPENDENCIES`.